rust/core-lib/src/editor.rs

```diff
@@ -169,7 +169,8 @@ impl<W: Write + Send + 'static> Editor<W> {
         self.views.len() > 0
     }
 
-    pub fn set_path<P: AsRef<Path>>(&mut self, path: P) {
+    /// should only ever be called from `BufferContainerRef::set_path`
+    pub fn _set_path<P: AsRef<Path>>(&mut self, path: P) {
```

Comment:
yes, exactly. It might be that the path just shouldn't be part of the editor, but I want to at least make it clear right now that setting path is not intended as real public API.